app/controllers/concerns/pubmed_search.rb

```diff
@@ -2,8 +2,8 @@ module PubmedSearch
   extend ActiveSupport::Concern
 
   def search_pubmed(search_term)
-    @search_terms = search_term.split.join("+")
-    search_url = 'http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&retmode=json&retmax=20&term='+@search_terms
+    @search_term = search_term.split.join("+")
+    search_url = 'http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&retmode=json&retmax=20&term='+@search_term
```

Comment:
I'm a little hesitant about having large hardcoded strings here. We might want to make a service object or something that represents the search functionality. It might look something like:

```
class SearchPubmed
  def init(options={})
    @base_url = 'http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi'

    # This joins the url parameters, like `retmode=json`
    options.map {|parameter, value| "&#{parameter}=#{value}" }.join
    options = "?" + options[1..-1]
  end

  def search(term)
    # Net/HTTP call goes here, appended to end of base url + options
  end
end